peer-ids/peer-ids.md

```diff
@@ -131,16 +131,43 @@ behavior.
 
 ### String representation
 
-Peer Ids are multihashes, and they are often encoded into strings.
-The canonical string representation of a Peer Id is a base58 encoding with
-[the alphabet used by bitcoin](https://en.bitcoinwiki.org/wiki/Base58#Alphabet_Base58).
-This encoding is sometimes abbreviated as `base58btc`.
+Peer Ids are [multihashes][multihash] represented with [CIDs](https://github.com/ipld/cid) when encoded into strings.
+
+CID is a multihash with a prefix that specifies things like base encoding, cid version and the type of data behind it:
+
+```
+<cidv1> ::= <multibase><cid-version><multicodec><multihash>
+```
+
+Encoding and decoding of string representation must follow [CID spec][cid-decoding].
 
-An example of a `base58btc` encoded SHA256 peer id: `QmYyQSo1c1Ym7orWxLYvCrM2EmxFTANf8wXmmE7DWjhx5N`.
+#### libp2p-key CID
```

Comment:
We may want to specify an explicit decoding algorithm:

1. If the peer ID starts with Q or 1, decode as a multihash.
2. Otherwise, decode as a CID.